CAIPs/caip-345.md

```diff
@@ -0,0 +1,104 @@
+---
+caip: CAIP-345
+title: Wallet Service URL property
+author: Chris Smith (@chris13524)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/discussions/345
+status: Draft
+type: Standard
+created: 2025-02-17
+updated: 2025-02-17
+requires: 25
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the CAIP.-->
+This CAIP defines a mechanism for CAIP-25 wallets to advertize support for a "wallet service" which can handle requests for specific wallet RPC methods instead of apps sending these requests directly to the wallet app.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+A short (~200 word) description of the technical issue being addressed.
+
+## Motivation
+<!--The motivation is critical for CAIP. It should clearly explain why the state of the art is inadequate to address the problem that the CAIP solves. CAIP submissions without sufficient motivation may be rejected outright.-->
+In protocols such as WalletConnect it is bad UX to redirect the user to a wallet to handle requests that are non-actionable to them. Examples of this include:
+- `wallet_getCapabilities` - is this real? point to EIP
+- `wallet_getCallsStatus` - point to EIP
+- `wallet_getAssets` - point to EIP
```

Comment:
```suggestion
- `wallet_getAssets` - Defined in [EIP-7811](https://eip.tools/eip/7811)
```

Note that this is linked to eip.tools instead of eip.ethereum.org, because the PR hasn't been merged yet.  We'll have to update it later, or else point to the open-PR's specific head commit, at time of publishing here if that's still the case.